packages/extension-store-locator/src/setup-server.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+    Application as ExpressApplication,
+    ApplicationExtension as ExpressApplicationExtension
+} from '@salesforce/pwa-kit-runtime/ssr/server/extensibility'
+import {ServerExtensionConfig as Config} from './types'
+
+class StoreLocatorExtension extends ExpressApplicationExtension<Config> {
+    extendApp(app: ExpressApplication): ExpressApplication {
+        return app
+    }
+}
+
+export default StoreLocatorExtension
```

Comment:
I've noticed some errors on MRT remotely - 

```
ERROR  pwa-kit-runtime.RemoteServerFactory._setupExtensions ERROR '@salesforce/extension-sample' is not a valid PWA-Kit Application Extension, please ensure you are exporting a class of type 'ApplicationExtension'. Skipping.
2024-10-28T17:28:10.077Z 536adf35 INFO   pwa-kit-runtime INFO Loading extension: @salesforce/extension-store-locator
2024-10-28T17:28:10.077Z 536adf35 ERROR  pwa-kit-runtime.RemoteServerFactory._setupExtensions ERROR '@salesforce/extension-store-locator' is not a valid PWA-Kit Application Extension, please ensure you are exporting a class of type 'ApplicationExtension'. Skipping.
2024-10-28T17:28:10.077Z 536adf35 INFO   pwa-kit-runtime INFO Loading extension: @salesforce/extension-sample-with-overrides
2024-10-28T17:28:10.078Z 536adf35 ERROR  pwa-kit-runtime.RemoteServerFactory._setupExtensions ERROR '@salesforce/extension-sample-with-overrides' is not a valid PWA-Kit Application Extension, please ensure you are exporting a class of type 'ApplicationExtension'. Skipping.
```

I'm not sure if there's anything we need to fix/change in the SDK. @bendvc 

It's not blocking.

Reply:
This code is probably far removed in my PR.. so it would be interesting to see if this still happens in my branch. If you are worried about that, create a small bug and we can either act on it if it still persists or close it out.